docker/CMSRucioClient/scripts/syncRolesToAttributesLocalUsers.py

```diff
@@ -93,18 +93,56 @@ def set_local_identities(client, site, dns=None, user_map=None, site_map=None):
         client.del_identity(account=account, identity=identity, authtype='X509')
 
 
+def set_group_identities(client, group, dns, dn_email_map):
+
+    account = group
+    email = account + "@cern.ch"
```

Comment:
I doubt this is right. The group names don't have CMS in them but I would bet the e-groups do. Waiting for Stefan to come back with an answer.

Reply:
Yes, my bad. I checked it before but against a different name. I have asked Stephan for a way, https://cms-talk.web.cern.ch/t/moving-cms-embedding-samples-run-2-ul-to-t1s-global-storage-area/10203/66?u=rchauhan

Reply:
Found it, it's not supposed to be `account + @cern.ch`. But instead, `egroup+@cern.ch`. 
This is what I tested, but messed up in the code. 

The reason is e-group information is not available at the "user" endpoint. 
Option 1: We do a call to the "group" endpoint (https://cms-cric.cern.ch/api/accounts/group/query/list/?json) and create a `group -> egroup` map. 
Option 2: Use the other PR, which is based completely on the "group" endpoint.